### PR TITLE
Wahlplattform Hotfix "The Morning After"

### PR DESCRIPTION
**Pull request by garamond**

### CODE REVIEW

File: components/Vote/text.js [hunk 1]
```diff
@@ -44,22 +45,32 @@ const PMedium = (props) =>
...
-        .map((c, i) =>
-          <PMedium key={i}>
-            <RawHtml dangerouslySetInnerHTML={{ __html: c }} />
-          </PMedium>
-        )
-    }
-  </div>
+export const Body = withInNativeApp(({ dangerousHTML, inNativeApp }) => {
+  const html = inNativeApp
+    ? dangerousHTML.replace(/'/g, '"').replace(/target="_blank" /, '')
```
Comment (tpreusse) [inline components/Vote/text.js]: RE `target="_blank" ` replace:
Needs `g` flag too, why with an space at the end? `<a href="x" target="_blank">` is valid too

File: components/Vote/text.js [hunk 2]
```diff
@@ -78,17 +89,23 @@ const PSmall = ({ children, indent = true }) =>
...
-        .map((c, i) =>
-          <PSmall key={i} indent={indent}>
-            <RawHtml dangerouslySetInnerHTML={{ __html: c }} />
-          </PSmall>
-        )
-    }
-  </div>
+export const Small = withInNativeApp(({ dangerousHTML, inNativeApp, indent = true }) => {
+  const html = inNativeApp
+    ? dangerousHTML.replace(/'/g, '"').replace(/target="_blank" /, '')
```
Comment (tpreusse) [inline components/Vote/text.js]: see above, maybe move to util fn

File: components/Vote/text.js (hunk 1 above)
Comment (garamond) [reply components/Vote/text.js]:
```suggestion
    ? dangerousHTML.replace(/'/g, '"').replace(/target="_blank"/g, '')
```

File: components/Vote/text.js (hunk 2 above)
Comment (garamond) [reply components/Vote/text.js]:
```suggestion
    ? dangerousHTML.replace(/'/g, '"').replace(/target="_blank"/g, '')
```

**Pull request merged**